tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/recommendation-intro-validator.js

```diff
@@ -0,0 +1,19 @@
+import ValidateContent from "../../../helpers/content-validators/content-validator.js";
+
+export const ValidateRecommendationIntro = (introPage, recommendationUrl) => {
+    const { slug, header, content } = introPage;
+    
+    cy.url().should("include", `${recommendationUrl}/${slug}`);
+
+    cy.get("h1.govuk-heading-xl").contains(header);
+ 
+    if (content) {
+        for (const component of content) {
+            if (!component) {
+                console.log(`content is missing in page.`, introPage);
```

Comment:
Is there a way here to not only log what page the content is missing from but the content itself?

Reply:
I think this actually checking for null content in the Contentful export, so probably just needs the message changing. I replicated this from ValidatePage so changing it there too.